content/200-orm/500-reference/050-prisma-client-reference.mdx

```diff
@@ -1293,7 +1293,7 @@ const updatedUserCount = await prisma.user.updateMany({
 ##### Update all `User` records where the `email` contains `prisma.io` and at least one related `Post` has more than 10 likes
 
 ```ts
-const deleteUser = await prisma.user.updateMany({
+const updateUser = await prisma.user.updateMany({
```

Comment:
Quick change to make your update even more correct 🙂 
```suggestion
const updatedUserCount = await prisma.user.updateMany({
```

Reply:
Haha it's true, this is a bulk update after all. Have a good day :)